services/app-api/serverless.yml

```diff
@@ -217,6 +206,31 @@ functions:
       - !Ref PrismaClientEngineLambdaLayer
       - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-18-1:1
 
+  graphql_external:
+    handler: src/handlers/apollo_gql.graphqlHandler
+    events:
+      - http:
```

Comment:
Could these events be inside the regular graphql lambda or do they need to be their own lambda?

Reply:
I confirmed it works with combining with the regular graphql lambda. I think this falls under "[lambda proxy integration](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html)" which is the default used with [serverless](https://www.serverless.com/framework/docs/providers/aws/events/apigateway#lambda-proxy-integration)